bin/dfx-sns-transfer-to-faucet

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+PATH="$SOURCE_DIR:$PATH"
+. "$SOURCE_DIR/versions.bash"
+
+# Source the clap.bash file ---------------------------------------------------
+source "$SOURCE_DIR/clap.bash"
+# Define options
+clap.define short=n long=network desc="The dfx network to use" variable=DFX_NETWORK default="local"
+# Source the output file ----------------------------------------------------------
+source "$(clap.build)"
+set -euo pipefail
+
+LEDGER_CANISTER_ID="$(dfx canister call sns_root list_sns_canisters '(record {})' | idl2json | jq -r '.ledger | .[]')"
```

Comment:
Please add `--network "${DFX_NETWORK}"` to the `dfx canister call` so that the same program can be used against testnets.

Likewise in the canister calls below and in `dfx-sns-disburse-neuron` above.

Reply:
I was following `bin/dfx-sns-sale-buy` and it also doesn't pass --network.
I assumed that dfx would just pick up the DFX_NETWORK set by `clap` although I just realized that dfx-sns-sale-buy does `export DFX_NETWORK` where this script doesn't.

So now it does. I assume that's as good as passing --network.

Reply:
I'm afraid that doesn't work.  buy does actually pass `--network` on lines 20, 25 and 32.  The last actually passes "local" as the network instead of `$DFX_NETWORK`.  That is a bug.

Reply:
Done. And also fixed the "local" in bin/dfx-sns-sale-buy.